.github/workflows/continuous-integration.yml

```diff
@@ -105,6 +105,7 @@ jobs:
       - name: Install petsc4py
         working-directory: ./petsc
         run: |
+          python -m pip install Cython
           PETSC_ARCH=linux-gnu-complex-64 python -m pip install src/binding/petsc4py
```

Comment:
I think we need to add here `PETSC_DIR=~/petsc-main`

Reply:
We are working in the directory `./petsc` here. I will try anyway

Reply:
The installation works after 0d589a6